### PR TITLE
63233: About Releases

### PR DESCRIPTION
**Pull request by jonquil2002**
SC: https://app.shortcut.com/replicated/story/63233/add-conceptual-topic-for-about-creating-releases

Preview: https://deploy-preview-811--replicated-docs.netlify.app/vendor/releases-about

### CODE REVIEW

File: docs/vendor/releases-about.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## Overview of Releases
+
+You can use the Replicated vendor portal to create and release versions of your application to various release channels. The vendor portal hosts a built-in YAML editor and linter to help you write and validate manifest files.
+
+Alternatively, you can use the replicated CLI and API to automate releases. For more information about using the CLI, see [Installing the replicated CLI](../reference/replicated-cli-installing). For information about the API, see [Using the Vendor API v3](..reference/vendor-api-using).
+
+Application files can be either Helm charts or Kubernetes manifest files, which can include standard manifests such as Deployment and Service resources.
+
+### Using Custom Resources
```
Comment (paigecalvert) [inline docs/vendor/releases-about.md:13]: Your call, I could go either way on this: Could consider removing the `-ing` verbs in these subheadings to be just "Custom Resources", "Release Channels", "Release Properties". (Might serve the same useful purpose of making the overview conceptual content more skimmable without needing language for specific actions that the user is taking related to these concepts)

**Review by paigecalvert — DISMISSED** [review]
Looks great! I added a few comments that are mostly just suggestions to take or leave. 

Other thing I saw was that it looks like this adds two images that are not in the topic. Did you end to delete these or to reference them in the topic?

File: docs/vendor/releases-about.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+This topic describes concepts about creating and promoting releases, editing release properties, and information about the Releases page in the Replicated vendor portal.
+
+## Overview of Releases
+
+You can use the Replicated vendor portal to create and release versions of your application to various release channels. The vendor portal hosts a built-in YAML editor and linter to help you write and validate manifest files.
+
+Alternatively, you can use the replicated CLI and API to automate releases. For more information about using the CLI, see [Installing the replicated CLI](../reference/replicated-cli-installing). For information about the API, see [Using the Vendor API v3](..reference/vendor-api-using).
+
+Application files can be either Helm charts or Kubernetes manifest files, which can include standard manifests such as Deployment and Service resources.
```
[the remark this answers is not in the document]
Comment (jonquil2002) [reply docs/vendor/releases-about.md:11]: Good question. I just checked and it looks like the Kubernetes Operators are only impacted by[ images and namespaces](https://docs.replicated.com/vendor/operator-packaging-about), and [do not use the Config screen option for the admin console](https://docs.replicated.com/vendor/operator-packaging-about).

**Comment by jonquil2002** [comment]
@paigecalvert Yes, I did add those two screenshots intentionally, at least for this revision. Thanks for the great feedback. Please re-review when you get a chance.

**Pull request merged**